test/test_crypto.py

```diff
@@ -33,53 +33,36 @@ class TestEncryptUtils(unittest.TestCase):
     
     def setUp(self):
         '''
-        Some of the functions require the certificate and key to be files.
+        If no key/cert pair found, generate a new
+        key/cert pair and store as a file.
         '''
-        self.certfile, self.certpath = tempfile.mkstemp(prefix='cert')
-        os.write(self.certfile, TEST_CERT)
-        os.close(self.certfile)
+        if not (os.path.exists(TEST_CERT_FILE) and os.path.exists(TEST_KEY_FILE)):
+            # create a key/cert pair
+            call(['openssl', 'req', '-x509', '-nodes', '-days', '365',
```

Comment:
Yeah, that's the shortest I think you can do without diving into madness/low-level options.